dataconnect/dataconnect/movie-connector/mutations.gql

```diff
@@ -61,14 +60,12 @@ mutation AddReview(
 
 # Update a user's review for a movie
 mutation UpdateReview(
```

Comment:
How should this be used at all? It's not called from the client app at all ever since it was introduced in 0a8a6806. Shall we merge `UpdateReview` back into `AddReview` for an upsert?

Reply:
That makes sense, it wasn't used, but I wanted an example for _update as well